build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
We should really have a comment at the top of this file saying what it is and that the version in master is the only applicable version. Otherwise people will try editing in other branches and that's going to surprise people.

I presume we are going to allow comments in this file. If somebody is going to play the "JSON doesn't allow comments" card then let's pick a different format.